runs/2017/pyladies-brno-podzim-po/info.yml

```diff
@@ -1,7 +1,7 @@
 title: Začátečnický kurz Pyladies
-subtitle: Brno - podzim 2017 - pondělí
+subtitle: Brno - jaro 2017 - pondělí
```

Comment:
Měníš podzimní yml file, ale titulek nastavuješ na jaro, nerozumím, co se děje.

Reply:
Málo kafe.
[Další commit](https://github.com/pyvec/naucse.python.cz/pull/189/commits/cf57f7fe16a3a11b3828ef3bef07bb1637bcf4fa) to revertuje.

Reply:
Posquashuješ to prosím?